src/frontend/src/flows/recovery/pickRecoveryDevice.ts

```diff
@@ -1,17 +1,17 @@
 import { html, render } from "lit-html";
 import { DeviceData } from "../../../generated/internet_identity_types";
 import { mainWindow } from "../../components/mainWindow";
+import { securityKeyIcon, seedPhraseIcon } from "../../components/icons";
 
 const pageContent = () => {
   const pageContentSlot = html`
-    <h1 class="t-title t-title--main">Choose your Recovery Method</h1>
-    <p class="t-paragraph t-lead">
-      How do you want to recover your Internet Identity?
-    </p>
-    <div class="l-stack">
-      <h2 class="t-title">Recovery devices</h2>
-      <ol class="c-list l-stack" id="deviceList"></ol>
-    </div>
+    <hgroup>
+      <h1 class="t-title t-title--main">Choose Recovery Method</h1>
+      <p class="t-paragraph t-lead">
+        How do you want to recover your Internet Identity?
+      </p>
+    </hgroup>
+    <div id="deviceList"></div>
```

Comment:
This changed from an `ol` to a `div`, just double checking that it's on purpose (since the id suggest it's a list)!

Reply:
But before it resulted in a list wrapped by an other list in the dom. the "DeviceList" is used as an ID to append the actual list, I think its fine.